__tests__/app.test.js

```diff
@@ -279,13 +279,52 @@ describe.only("GET /api/reviews", () => {
       .get("/api/reviews?sort_by=katherine")
       .expect(400)
       .then((response) => {
-        expect(response.body.msg).toBe("Bad request.");
+        expect(response.body.msg).toBe("Invalid sort query");
+      });
+  });
+
+  test("200: returns an array of items provided in asc or desc order when provided valid order arg", () => {
+    return request(app)
+      .get("/api/reviews?order=ASC")
+      .expect(200)
+      .then((response) => {
+        const { reviews } = response.body;
+        expect(reviews).toBeSorted({ descending: false, key: "created_at" });
+      });
+  });
+
+  test("400: bad request when order not valid (not ASC or DESC)", () => {
+    return request(app)
+      .get("/api/reviews?order=KATHERINE")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Invalid order query");
+      });
+  });
+
+  test("200: filters by category when provided with valid category", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then((response) => {
+        const { reviews } = response.body;
+        expect(reviews.length).toBe(1);
+        reviews.forEach((user) => {
+          expect(user).toMatchObject({
+            category: "dexterity",
+          });
+        });
+      });
+  });
+  test("400: If given invalid category to filter by, should return error message", () => {
```

Comment:
Id suggest this is 404 -the category is valid, but not found. 
400/invalid would be a number for example